specification/cognitiveservices/data-plane/LUIS/Authoring/preview/v3.0/LUIS-Authoring.json

```diff
@@ -7862,6 +7890,256 @@
       }
     }
   },
+  "x-ms-paths": {
+    "/apps/import?JsonVersion=V2": {
+      "post": {
+        "description": "Imports an application to LUIS, the application's structure is included in the request body.",
+        "operationId": "Apps_ImportV2App",
```

Comment:
Is V2 a term that will be advertised to customer in documentation, or just a way to get a different name in the SDK and avoid conflict?

Reply:
V2 is referring to the prior API version
Importing method input is different in V2 than V3

And we need to support both in V3, therefore, this naming is to avoid conflict with importing a V3 app, as well to still support importing old V2 app, in order to solve a bug for the SDK users.